docs/streams/stream-examples/rest-apis-stream-example.md

```diff
@@ -245,9 +243,9 @@ import time
 
 # Constants
 URL = "api-play.paas.macrometa.io"
-HTTP_URL = f"https://{URL}"
+HTTP_URL = "https://{URL}"
```

Comment:
f is needed here
```suggestion
HTTP_URL = f"https://{URL}"
```

Reply:
More about f-strings can be read here : https://www.geeksforgeeks.org/formatted-string-literals-f-strings-python/